odrive/configs/MX_M2.json

```diff
@@ -0,0 +1,499 @@
+{
+    "axis0": {
```

Comment:
what MX?

Reply:
we don't know yet, depends on what ends up going on that odrive